### PR TITLE
(GNOME) Palette

### PR DESCRIPTION
**Pull request by ZanderBrown**
https://gitlab.gnome.org/World/palette/

### CODE REVIEW

File: org.gnome.zbrown.Palette.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
```
Comment (TingPing) [inline org.gnome.zbrown.Palette.json]:
```suggestion
```

Default.

File: org.gnome.zbrown.Palette.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        "cxxflags": "-O2 -g",
```
Comment (TingPing) [inline org.gnome.zbrown.Palette.json]:
```suggestion
```

File: org.gnome.zbrown.Palette.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        "env": {
+            "V": "1"
```
Comment (TingPing) [inline org.gnome.zbrown.Palette.json]: Just remove this `env`, you don't even use autotools here.

File: org.gnome.zbrown.Palette.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        }
+    },
+    "modules": [
+        {
+            "name": "palette",
+            "builddir": true,
```
Comment (TingPing) [inline org.gnome.zbrown.Palette.json]:
```suggestion
```

default

**Comment by ZanderBrown** [comment]
@TingPing may I suggest you propose some of these changes to gnome-builders default manifest?

**Comment by nedrichards** [comment]
bot, build org.gnome.zbrown.Palette"

**Comment by nedrichards** [comment]
bot, build org.gnome.zbrown.Palette

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.gnome.zbrown.Palette

**Comment by ZanderBrown** [comment]
:tada: thanks @nedrichards 

Seems I don't have write access though?

**Pull request closed without merging**